Any negative number is meaningless to 'maxTimeDiffSeconds'

## Question

I set '-2' into option 'maxTimeDiffSeconds'. It thrown exception.

```
[ERROR] 2020-10-03 01:05:10,009 --javaDataflowElasticJob_Worker-1-- [org.apache.shardingsphere.elasticjob.error.handler.general.LogJobErrorHandler] Job 'javaDataflowElasticJob' exception occur in job processing 
org.apache.shardingsphere.elasticjob.infra.exception.JobExecutionEnvironmentException: Time different between job server and register center exceed '0' seconds, max time different is '-2' seconds.
	at org.apache.shardingsphere.elasticjob.lite.internal.config.ConfigurationService.checkMaxTimeDiffSecondsTolerable(ConfigurationService.java:103) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.checkJobExecutionEnvironment(LiteJobFacade.java:81) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:85) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:35) [classes/:na]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202) [quartz-2.3.2.jar:na]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:na]
```

--- a/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/config/ConfigurationService.java
+++ b/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/config/ConfigurationService.java
@@ -94,7 +94,7 @@ public final class ConfigurationService {
      */
     public void checkMaxTimeDiffSecondsTolerable() throws JobExecutionEnvironmentException {
         int maxTimeDiffSeconds = load(true).getMaxTimeDiffSeconds();
-        if (-1 == maxTimeDiffSeconds) {
+        if (0 > maxTimeDiffSeconds) {
             return;
         }
         long timeDiff = Math.abs(timeService.getCurrentMillis() - jobNodeStorage.getRegistryCenterTime());

